src/asana/helpers.py

```diff
@@ -424,7 +454,7 @@ def _task_description_from_pull_request(pull_request: PullRequest) -> str:
         + author
         + _generate_assignee_description(pull_request.assignee())
         + _wrap_in_tag("strong")("\n\nDescription:\n")
-        + convert_urls_to_links(escape(pull_request.body(), quote=False))
```

Comment:
FYI @arvindvjkumar  -- I believe this change will take care of making sure the @-mentioned people in the task descriptions will get converted to Asana mentions